File: word-break.java
public void convertHTMLtoPDF() throws IOException {
    FontProvider fontProvider = new FontProvider();
    FontProgram fontProgram = FontProgramFactory.createFont("NotoSansJP-Medium.otf");
    fontProvider.addFont(fontProgram);
    ConverterProperties properties = new ConverterProperties();
    FontProvider fontProvider = new DefaultFontProvider(true, true, true);
    properties.setFontProvider(fontProvider);
    HtmlConverter.convertToPdf(new FileInputStream("HTML/input_html.html"), new FileOutputStream("output_file.pdf"), properties);
}

File: word-break.cs
public virtual void ConvertHTMLtoPDF() {
    FontProvider fontProvider = new FontProvider();
    FontProgram fontProgram = FontProgramFactory.CreateFont("NotoSansJP-Medium.otf");
    fontProvider.AddFont(fontProgram);
    ConverterProperties properties = new ConverterProperties();
    FontProvider fontProvider = new DefaultFontProvider(true, true, true);
    properties.SetFontProvider(fontProvider);
    HtmlConverter.ConvertToPdf(new FileInputStream("HTML/target-counter.html"), new FileOutputStream("output_file.pdf"), properties);
}
